spec/models/like_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Validations For the Like model' do
+    before(:each) do
+      @like = Like.new(author_id: 4, post_id: 10)
+    end
+
+    before { @like }
+
+    it 'when author_id is defined' do
+      @like.author_id = false
+      expect(@like).to_not be_valid
+    end
+
+    it 'when post_id is defined' do
+      @like.post_id = nil
+      expect(@like).to_not be_valid
+    end
+  end
+end
```

Comment:
- [ ] Please make sure to test all methods implemented in the `Like` model, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods) requirement.